ballsdex/core/bot.py

```diff
@@ -304,13 +302,15 @@ async def on_ready(self):
             await self.add_cog(Dev())
 
         loaded_packages = []
-        for package in PACKAGES:
+        for package in settings.packages:
+            package_name = package.split(".")[-1]
```

Comment:
I like this better, it doesn't hide the full path for custom packages
```suggestion
            package_name = package.replace("ballsdex.packages.", "")
```